back-end/stagram/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+# Create your models here.
+
+
+class User(models.Model):
+    username = models.CharField(max_length=128)
+    password = models.CharField(max_length=25)
+    email = models.CharField(max_length=50)
+
+    def __str__(self):
+        return f'{self.username}-{self.email}'
```

Comment:
We do not want to create a `User` model since one already exists: https://docs.djangoproject.com/en/4.0/ref/contrib/auth/
So you can remove this model and instead `from django.contrib.auth import User` (or something to that effect)

Reply:
I deleted the User model